src/styles/main.scss

```diff
@@ -2,3 +2,269 @@
 @import "partials/partials";
 @import "elements/elements";
 
+/* Space out content a bit */
+body {
+  padding-top: 49px;
+  /* padding-bottom: 20px; */
+
+}
+
+/* Everything but the jumbotron gets side spacing for mobile first views */
+.header,
+.marketing,
+.footer {
+  padding-right: 15px;
+  padding-left: 15px;
+}
+
+/* Custom page header */
+.header {
+  padding-bottom: 20px;
+  border-bottom: 1px solid #e5e5e5;
+}
+/* Make the masthead heading the same height as the navigation */
+.header h3 {
+  margin-top: 0;
+  margin-bottom: 0;
+  line-height: 40px;
+}
+
+/* Custom page footer */
+.footer {
+  padding: 20px 30px;
+  font-size:18px;
+  color: #777;
+  border-top: 3px solid #ddd;
+  background:#e1e1e1;
+}
+
+
+.navbar {
+	background: #032e59;
+}
+.navbar img {
+	margin-top: -7px;
+}
+
+
+.now-next {
```

Comment:
@LordAro @ChrisTheBaron So in GO we refer to it as current and next but in some other things it's now and next, think the api calls it current and next, shall we stick to that or?

If so we should probably change this..